maas/resource_maas_machine_test.go

```diff
@@ -0,0 +1,85 @@
+package maas_test
```

Comment:
This is not needed at all. Please remove the entire file. Or more precisely, we have to add `maas_machine` acceptance tests but for the means of `maas_machines` data source they are not necessary. The reasoning is explained in the comment above.